Makefile

```diff
@@ -280,15 +296,27 @@ bin/helm:
 	tar -zxvf $(TMP)/$(HELM_RELEASE) -C $(TMP)
 	mv $(TMP)/linux-amd64/helm bin/helm
 
-.PHONY: values.yml
-values.yml:
+bin/dep:
+	mkdir -p bin
+	curl -L https://github.com/golang/dep/releases/download/v0.5.0/dep-linux-amd64 > bin/dep
+	chmod +x bin/dep
+
+
+versions-values.yaml:
```

Comment:
The idea is to not have the ruby script merge the info from the `versions.yml` file into the `values.yaml` file. If the command gets run consecutively (which is likely if it is required by the tests), then it will just keep appending the same information to the `values.yaml` file. Doing this makes it so we have to pass in the `versions.yml` info as a separate file, but it works better from a make target perspective.